src/js/components/molecules/list-section/list-section.scss

```diff
@@ -75,7 +75,7 @@
 }
 
 .list-section__item {
-    margin-bottom: $buildingUnit;
+    margin-bottom: 32px;
```

Comment:
Idem: $buildingUnit * ?

Het is sowieso een rare meeteenheid?

Reply:
Done